lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -37,6 +37,10 @@ html.video-fullscreen{
       color: $white;
     }
   }
+
+  @media print {
```

Comment:
I don't love this rule, because it obscures completely that you've printed from a staff user account.